jwst/wfs_combine/wfs_combine_step.py

```diff
@@ -25,11 +25,14 @@ class WfsCombineStep(Step):
         suffix = string(default="wfscmb")
     """
 
+    def make_output_path(self, basepath, *args, **kwargs):
```

Comment:
So where does `make_output_path` get used? I don't see it called from this code module anywhere. Or does it replace/override a version of the same function that exists in stpipe?

Reply:
Thanks for taking a look. It's used by stpipe:
https://github.com/spacetelescope/stpipe/blob/d1a0a6d25de45400e315afa4fa3cee8633a6b545/src/stpipe/step.py#L1039
The usage here overrides the property which returns a partial.

Reply:
OK, that's what I figured. Question resolved ...